tests/functional/endpoints.ts

```diff
@@ -113,6 +112,25 @@ describe('Health Endpoint Returns the Correct Fields', async (assert: any) => {
     })
   }
 
+  {
+    let metricsData
+
+    try {
```

Comment:
there is any real reason to fail?

Reply:
I think is better to avoid the block `try/catch` in the tests, sometimes brings problems with false positives.

Reply:
Check out this article for a good pattern for try blocks with riteway: https://medium.com/javascript-scene/rethinking-unit-test-assertions-55f59358253f

Reply:
thanks for sharing @warrenv I will review!

Reply:
@jwicks31 PR looks good, but I'd avoid `try` and give `Try` a shot if possible.

Reply:
Ok. I just removed the try block completely, since I don't think it was really necessary anyway. We aren't using them in other areas that are similar.